roles/nvidia_cuda/tasks/install-ubuntu.yml

```diff
@@ -4,25 +4,22 @@
     repo: ppa:graphics-drivers/ppa
     state: absent
 
-- name: Ubuntu | add pin file
-  copy:
-    src: "cuda-ubuntu.pin"
-    dest: "/etc/apt/preferences.d/cuda-repository-pin-600"
-    owner: "root"
-    group: "root"
-    mode: "0644"
-
-- name: Ubuntu | add key
+- name: Ubuntu | ensure old key is absent
```

Comment:
Do we want to eventually remove this or does it stay here forever?

Reply:
Unclear. In principle I think this could be removed once we were confident all existing installs were updated. But it's hard to know when that is. 🤷🏻‍♂️ 

If I were being ultra-conservative, I'd say this could be removed when the [last released drivers signed by the old key are EOL](https://docs.nvidia.com/datacenter/tesla/drivers/#cuda-drivers) (r470, July 2024). We might be able to come up with a faster schedule than that though.

However, I don't think this task is doing any *harm* (apart from code clutter). So I don't think there would be any rush to remove this.